pkgs/development/python-modules/seaborn/default.nix

```diff
@@ -49,8 +50,16 @@ buildPythonPackage rec {
   ] ++ lib.optionals (!stdenv.hostPlatform.isx86) [
     # overly strict float tolerances
     "TestDendrogram"
+  ] ++ lib.optionals stdenv.isDarwin [
+    # Requires internet connection, and, when it's available without darwin
+    # sandbox, throws:
+    #   OSError: [Errno 30] Read-only file system: '/homeless-shelter'
+    "test_load_dataset_string_error"
   ];
 
+  # Avoid using MacOSX backend to prevent crashing in darwin sandbox
+  ${if stdenv.isDarwin then "MPLBACKEND" else null}="Agg";
```

Comment:
Could this be converted to something in `preConfigure` or `preCheck`?

Reply:
Yes, but why?

Reply:
That was a trick to avoid a mass-rebuild that I expected. It should be perfectly fine to enable the Agg backend unconditionally.